### PR TITLE
Add stale bot config

### PR DESCRIPTION
**Pull request by ItamarShDev**
implementation for issue #2015

### CODE REVIEW

**Comment by ItamarShDev** [comment]
@xaviergonz @FredyC @mweststrate Lets define the best configs for the bot and merge it!

File: .github/stale.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 10
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+  - discuss
```
Comment (FredyC) [inline .github/stale.yaml]: We don't have these three labels. Do we need them? Perhaps the `discuss` one, otherwise I would remove these from here.

**Comment by FredyC** [comment]
It LGTM like that, we can tweak it over time if necessary.

The React repo has some additional feature of auto locking stale issues after bot closes them. It's useful so people don't tend to spam into dead issues but create a new one with a link to old one if relevant. Strangely enough, I don't see such option [in the configuration](https://github.com/probot/stale#usage).

File: .github/stale.yaml (hunk 1 above)
Comment (ItamarShDev) [reply .github/stale.yaml]: will remove

**Comment by ItamarShDev** [comment]
@FredyC Can I merge to master?

**Comment by ItamarShDev** [comment]
@xaviergonz i have set the days to a higher value the in MST.
seems reasonable to set it to the same values as in MST

your thoghts?

**Comment by xaviergonz** [comment]
Up to you, I set those because I thought 2 weeks gives people plenty of time to interact if they are serious into getting something fixed.

I'd also revamp the labels and make sure the ones that don't have to be stale are on the exemption list, since once stale is enabled it will start closing old issues as crazy

**Comment by FredyC** [comment]
> I'd add only: issues so it doesn't auto close prs

What's wrong about auto closing PRs? I mean if it's valuable PR, we can pin, but if it's stale because the author decided not to continue on it, there is no reason to keep it open, right? Perhaps for that purpose, we should have a `pinned` label after all or something similar. 

> I'd also revamp the labels and make sure the ones that don't have to be stale are on the exemption list, since once stale is enabled it will start closing old issues as crazy

I would keep these labels as exempt

```
breaking-change
bug
candidate-for-next-version
discuss
duplicate
enhancement
has PR
help wanted
needs investigation
pinned
ready-to-be-implemented-by-volunteer
waiting-for-standardized-decorators
```

**Comment by xaviergonz** [comment]
Sure, but make sure every open issue and pr has a label before the bot is enabled or it will be too late (talking from experience 😇)

**Comment by ItamarShDev** [comment]
@FredyC @xaviergonz updated according to the middle ground (imo)
didn't add PRs, maybe we need to have a test run without it, and add PRs if we see fits.

**Comment by FredyC** [comment]
I wouldn't worry about PRs much really, we have only 3 (besides this one) and I've pinned one. The other two are probably stale candidate anyway.

Nonetheless, even a closed PR can be reopened, so it's not a big deal.

**Comment by ItamarShDev** [comment]
@FredyC 
good.
awaiting merging permission :)

File: .github/stale.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  - bug
+  - has PR
+  - discuss
+  - enhancment
+  - candidate-for-next-version
+  - ready-to-be-implemented-by-volunteer
+  - needs investigation
+  - breaking-change
+  - bug
+  - duplicate
```
Comment (FredyC) [inline .github/stale.yaml]: Ah, this one is probably nonsense, I missed it :)

File: .github/stale.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 14
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 4
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - bug
+  - has PR
+  - discuss
+  - enhancment
```
Comment (FredyC) [inline .github/stale.yaml]:
```suggestion
  - enhancement
```

**Comment by FredyC** [comment]
I wonder when the magic starts happening, would be expecting something by now.

**Pull request merged**